src/loaders/IVolumeLoader.ts

```diff
@@ -6,6 +6,7 @@ export class LoadSpec {
   scene = 0;
   time = 0;
   // sub-region; if not specified, the entire volume is loaded
+  // specify as floats between 0 and 1
```

Comment:
Just for the record, I still feel like in a LoadSpec, this should be an explicit pixel value and not a relative fraction.

Reply:
I'd certainly be in favor of adding a field to `LoadSpec` that specifies when the values are in pixels, but I don't know how a pixel value would work when the loader is free to choose its own scale level. Would it be based on pixels of the highest-resolution scale level?

Reply:
if a loadSpec's subregion were always in xyz pixels, it would have to always be consistent with the same loadspec's multiresolutionLevel.    Changing levels would require you to calculate the fractional values, and then rescale them to the new level's pixel dimensions.  (Similar to how you resolve from fraction to pixels in order to actually load the data)

Reply:
you could still have a "null" state for the region that just says "give us the whole extents by default"

Reply:
Maybe figuring out how to populate the initial loadspec is a thing -- you'd want to be able to ask  for some dimension info in order to know the ranges you're dealing with.

Reply:
One more thought - if a loader is auto-selecting a multiresolution level, then it has to modify the loadspec in some way before sending the load requests along to the worker.  (It doesn't have to modify it permanently, but it could!)
But the initial incoming loadspec would still specify something explicit.